config.yml

```diff
@@ -30,94 +53,38 @@ workflows:
           requires:
             - orb-tools/pack
 
-      # trigger an integration workflow to test the dev version of your orb
-      # an SSH key must be stored in your orb's repository and in CircleCI
-      # (add the public key as a read/write key on GitHub; add the private
-      # key in CircleCI via SSH Permissions, with github.com as Hostname)
-      - orb-tools/trigger-integration-workflow:
+      # trigger an integration workflow to test the
+      # dev:${CIRCLE_SHA1:0:7} version of your orb
+      - orb-tools/trigger-integration-tests-workflow:
           name: trigger-integration-dev
-          ssh-fingerprints: <orb-fingerprint>
           requires:
             - orb-tools/publish-dev
-          filters:
-            branches:
-              ignore: master
 
-      # by default, the 1st job (above) will trigger only integration tests;
-      # the 2nd job (below) may also publish a production orb version
-      - orb-tools/trigger-integration-workflow:
-          name: trigger-integration-master
-          cleanup-tags: true
-          ssh-fingerprints: <orb-fingerprint>
-          tag: master
+  # This `integration-tests_prod-release` workflow will only run
+  # when the run-integration-tests pipeline parameter is set to true.
+  # It is meant to be triggered by the "trigger-integration-tests-workflow"
+  # job, and run tests on <your orb>@dev:${CIRCLE_SHA1:0:7}.
+  integration-tests_prod-release:
+    when: << pipeline.parameters.run-integration-tests >>
+    jobs:
+      # your integration test jobs go here: essentially, run all your orb's
+      # jobs and commands to ensure they behave as expected. or, run other
+      # integration tests of your choosing
+
+      # an example job
+      - integration-test-1
+
+      # publish a semver version of the orb. relies on
+      # the commit subject containing the text "[semver:patch|minor|major|skip]"
+      # as that will determine whether a patch, minor or major
+      # version will be published or if publishing should
+      # be skipped.
+      # e.g. [semver:patch] will cause a patch version to be published.
+      - orb-tools/dev-promote-prod-from-commit-subject:
+          orb-name: <orb-namespace>/<orb-name>
+          <orb-config-opts>
```

Comment:
It looks like this is causing issues with the linter (https://circleci.com/gh/CircleCI-Public/orb-starter-kit/299), might prefixing a "#" to make it a comment help?

Reply:
The orb starter kit shouldn't be followed on CircleCI, I'm not sure who added it. This config will never be linted until after it has already been replaced on the customer end.

Reply:
Thanks for the clarification!